### PR TITLE
Docusaurus fixes

### PR DESCRIPTION
**Pull request by nicolasochem**
Also re-set README as it was. It's ok to have duplication for now, let's come up with a strategy to migrate README to docusaurus once we have something.

### CODE REVIEW

File: docs/sidebars.js [hunk 1]
```diff
@@ -7,21 +7,21 @@ const sidebars = {
...
   tezosK8sSidebar: [
     'index',
+    'Prerequisites',
     {
       type: 'category',
       label: 'Common use cases',
       collapsed: false,
       items: [
         'Private-Node',
         'Private_Chain',
```
Comment (harryttd) [inline docs/sidebars.js:17]: This is using `_` and makes the url of the page inconsistent with the other pages that all use `-`

File: docs/RPC-Auth.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 You can optionally spin up an RPC authentication backend allowing trusted users to make RPC requests to your cluster.
 
-Follow the steps at `rpc-auth/README.md`.
+Follow the steps at [rpc-auth/README.md](https://github.com/oxheadalpha/tezos-k8s/blob/master/rpc-auth/README.md).
 
 # Using a custom Tezos build
```
Comment (harryttd) [inline docs/RPC-Auth.md:7]: This whole section `# Using a custom Tezos build` is not part of the RPC auth section in the readme. It is a separate topic.

File: docs/Private_Chain.md [hunk 3]
```diff
@@ -79,13 +79,13 @@ perform the following tasks:
...
 ```shell
-kubectl -n oxheadalpha get pods -l appType=tezos-node
+kubectl -n oxheadalpha get pods -l appType=octez-node
 ```
 
 You can view (and follow using the `-f` flag) logs for your node using the following command:
 
 ```shell
-kubectl -n oxheadalpha logs -l appType=tezos-node -c tezos-node -f --prefix
+kubectl -n oxheadalpha logs -l appType=octez-node -c tezos-node -f --prefix
```
Comment (harryttd) [inline docs/Private_Chain.md:88]: all `-c tezos-node` also needs to be `-c octez-node`

File: docs/Private-Node.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Running this results in:
 You can find your node in the oxheadalpha namespace with some status information using `kubectl`.
 
 ```shell
-kubectl -n oxheadalpha get pods -l appType=tezos-node
+kubectl -n oxheadalpha get pods -l appType=octez-node
```
Comment (harryttd) [inline docs/Private-Node.md:22]: This file should be named Public-Node as that is what is being described

File: docs/Private_Chain.md [hunk 5]
```diff
@@ -79,13 +79,13 @@ perform the following tasks:
 You can find your node in the oxheadalpha namespace with some status information using kubectl.
```
Comment (harryttd) [inline docs/Private_Chain.md:79]: File should be named `Private-Chain` to be consistent with all of the other files and resulting urls

File: README.md [hunk 6]
```diff
@@ -37,361 +37,100 @@ Follow the prerequisites step first. Then you can jump to either [joining mainne
...
 NOTE: You do not need to clone this repository! All necessary components will be installed.
 
-## Tezos k8s Helm Chart
-
-To add the Tezos k8s Helm chart to your local Helm chart repo, run:
-
-```shell
-helm repo add oxheadalpha https://oxheadalpha.github.io/tezos-helm-charts/
-```
+## Prerequisites
```
Comment (harryttd) [inline README.md:40]: I think we are still missing link to docs

Also, the table of contents needs to be updated.

Comment (harryttd) [reply README.md:40]: All this needs to be removed / updated as it doesn't apply anymore: https://github.com/oxheadalpha/tezos-k8s/pull/462/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R27-R38

```
# Tezos k8s
This README walks you through:
- spinning up Tezos nodes that will join a public chain, e.g. mainnet.
- creating your own Tezos based private blockchain.
Using `minikube`, your nodes will be running in a peer-to-peer network inside of a Kubernetes cluster. With your custom private blockchain, your network will be also running over a Zerotier VPN.
Follow the prerequisites step first. Then you can jump to either [joining mainnet](#joining-mainnet) or [creating a private chain](#creating-a-private-blockchain).

NOTE: You do not need to clone this repository! All necessary components will be installed.
```

File: README.md [hunk 7]
```diff
@@ -37,361 +37,100 @@ Follow the prerequisites step first. Then you can jump to either [joining mainne
...
 
-# Joining Mainnet
+- python3 (>=3.7)
+- [docker](https://docs.docker.com/get-docker/)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/)
+- [minikube](https://minikube.sigs.k8s.io/docs/)
+- [helm](https://helm.sh/)
+- A [ZeroTier](https://www.zerotier.com/) network with api access token
```
Comment (harryttd) [inline README.md:47]: optional

File: docs/values_to_doc.sh [hunk 8]
```diff
@@ -2,7 +2,7 @@
 
 extract_markdown() {
   uncommented=false
-  while IFS= read -r line; do
+  echo "$1" | while IFS= read -r line; do
```
[the remark this answers is not in the document]
Comment (nicolasochem) [reply docs/values_to_doc.sh]: No reason. I changed it.

File: docs/sidebars.js (hunk 1 above)
Comment (nicolasochem) [reply docs/sidebars.js]: I fixed this in another commit, I think?

File: docs/Private_Chain.md (hunk 3 above)
Comment (nicolasochem) [reply docs/Private_Chain.md]: thanks, done

File: README.md (hunk 6 above)
Comment (nicolasochem) [reply README.md:40]: @harryttd I reset the README to how it was before merging the docusaurus PR. We can have duplication for now and have a better strategy to move most content in docusaurus later.

File: docs/Private-Node.md (hunk 4 above)
Comment (nicolasochem) [reply docs/Private-Node.md:22]: done

File: docs/RPC-Auth.md (hunk 2 above)
Comment (nicolasochem) [reply docs/RPC-Auth.md]: I switched the README to how it was before, and removed this section.

**Review by harryttd — APPROVED** [review]
Thank you :)

**Pull request merged**